### PR TITLE
Change new event template to use forms

### LINKED ISSUE #197: Add template for new event
Already [added an issue template for adding new resources](https://github.com/ACCESS-Hive/access-hive.github.io/pull/196), but this isn't so relevant to new events, which in any case are a little trickier to add because it involves making a correctly formatted [YaML file](https://yaml.org/spec/1.2.2/).

It would be good to have a specific template for new events, particularly as it may lower the bar for people to suggest events to add, who might otherwise not be willing/able to do it themselves.

### PR DESCRIPTION
**Pull request by aidanheerdegen**
Change to use GitHub forms for event template.

Fixes #197

### CODE REVIEW

**Comment by aidanheerdegen** [comment]
Bit hard to see how this functions without accepting the PR, but does that look ok to you @ccarouge?

File: .github/ISSUE_TEMPLATE/new-event.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      description: Website URL for more information
+    validations:
+      required: false
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: |
+        Description of your event. Include enough information to be 
+        useful to potential attendees, but note will be truncated to
```
Comment (ccarouge) [inline .github/ISSUE_TEMPLATE/new-event.yml:57]:
```suggestion
        useful to potential attendees, but note it will be truncated to 
```

**Review by ccarouge — APPROVED** [review]
Looks great, just a minor suggestion.

**Comment by aidanheerdegen** [comment]
I'll merge. 🤞

**Pull request merged**